covid_moonshot/analysis/plots.py

```diff
@@ -251,6 +253,67 @@ def plot_convergence(
     return fig
 
 
+def plot_poor_convergence_fe_table(
+    runs: List[Run], energy_cutoff_kcal: float = 1.0,
+) -> plt.Figure:
+    """
+    Plot table of poorly converging free energy estimates with GEN
+
+    Parameters
+    ----------
+    runs : list of Run
+        Relative free energies (in kT)
+    energy_cutoff_kcal : float
+        Cutoff to consider a result as poorly converged (in kcal/mol)
+
+    Returns
+    -------
+    Figure
+        Figure containing the plot
+
+    """
+
+    complex_phases = [run.analysis.complex_phase for run in runs]
+    job_ids = [run.details.JOBID for run in runs]
+
+    std_dev_store = []
+    jobid_store = []
+
+    for run in runs:
+
+        try:
+
+            complex_gens = run.analysis.complex_phase.gens
+            std_dev = np.std([gen.free_energy.delta_f for gen in complex_gens])
```

Comment:
nit: do we want to skip the run if one of the gens is missing a free energy? If not, what about something like
```suggestion
            std_dev = np.std([gen.free_energy.delta_f for gen in complex_gens if gen.free_energy is not None])
```
(and removing the surrounding `try`)

Reply:
Yeah good point, we don't want to skip a whole run! This is a neater way of doing it.